### DISCUSSION TITLE
Could Photoview notify about similar photos?

### DISCUSSION THREAD

**Discussion by standin000** [photoview/photoview#1022] [Ideas]
I know synology photos has this function, it is useful for me collect lots of image from different people. thanks!

**Comment by kkovaletp** [comment]
@standin000, thanks for the idea.
Currently, we don't have this functionality, but I'd like to have it in the future.

How would you like to configure it in settings? What level of similarity would be enough to notify you?
How would you like to get these notifications in the UI? What behavior would you expect in the case of 2 images being reported to be similar? What if there are more than 2?
Would you like to have this functionality also for videos? How it should work in this case?

**Comment by standin000** [comment]
I just want there is find similar photos button, it will search similar photos in current folder and list them one by one, I decide which one to be deleted. 
I can not answer what level of similarity and similar video is too difficult and not necessary.

**Comment by kkovaletp** [comment]
Thanks for your details.
I think that searching just inside 1 folder on demand would be too specific, as each user has a different source media gallery structure.
Also, there are rare cases of having 100% identical files in the media gallery. In most cases, there will be several similar images in different resolutions/formats/geometry, etc., located in different folders.

To address these cases, from my point of view, it looks better to extend our scanner by adding a job to search for similar media. To make it work, we also need to publish all needed settings on the Settings page. As the scanner runs in the background, it will search for duplicated media at the run time, so the results should be accessible in the UI somewhere, and this is an open question: how to better add these results intuitively?
